conda_smithy/feedstocks.py

```diff
@@ -84,7 +84,7 @@ def clone_feedstock(feedstock_gh_repo, feedstocks_dir):
     if not os.path.exists(clone_directory):
         print('Cloning {}'.format(repo.name))
         new_repo = Repo.clone_from(repo.clone_url, clone_directory)
-        clone.delete_remote('origin')
+        new_repo.delete_remote('origin')
```

Comment:
Maybe it would just be better to make `new_repo` -> `clone` in both cases. It would be overwritten in the end, but by nothing different.